engine/src/main/resources/application.conf

```diff
@@ -36,6 +36,9 @@ system {
 
   // If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
   abort-jobs-on-terminate = false
+
+  // Max number of retries per job that the engine will attempt in case of a retryable failure received from the backend
+  maxRetries = 3
```

Comment:
I guess it doesn't hurt to have that threshold, although in the case of JES for example where this is used for preemptible VMs, the backend decides how many times a job needs to be retried if it gets preempted, based on the runtime attributes. So a WDL task with `preemptible: 5` runtime attribute running on JES, would actually only be retried 3 times if it runs on a Cromwell configured as above.